blocks/melody/ff_container/ff_container-expandable/ff_container-expandable.less

```diff
@@ -1,29 +1,36 @@
-.ff_container-expandable__dropdown {
-	padding: @ff_size_spacing_med;
-	border: 1px solid @ff_colour_melody_int_blue;
-	color: @ff_colour_melody_int_blue;
-	cursor: pointer;
-	background: white;
-}
+.ff_container-expandable {
+	&__header {
+		padding: @ff_size_spacing_med;
+		border: 1px solid @ff_colour_melody_int_blue;
+		color: @ff_colour_melody_int_blue;
+		cursor: pointer;
+		background: white;
+	}
 
-.ff_container-expandable__dropdown--small-screens {
-	@media all and (min-width: @ff_bp_medium) { 
-		display: none;
+	&__icon {
+		width: 1.5em;
+		height: 1.5em;
+		float: right;
 	}
-}
 
-.ff_container-expandable__icon  {
-	width: 1.5em;
-	height: 1.5em;
-	float: right;
-}
+	&__content {
+		max-height: 1000px;
```

Comment:
From my understanding, in order to animate the height of an element in CSS (to create a slide-up/slide-down) you must set either a height or a max-height on the element. It can't animate from 0 to auto(this is true for other properties as well). This could have been done in JQuery with a slideDown/SlideUp but we decided to use a CSS animation which would be less expensive. I am not an expert when it comes to CSS animations but I think this is the reason.